src/pages/CheckoutInfo.page.js

```diff
@@ -0,0 +1,23 @@
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+export class CheckoutInfoPage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+
+    continueSelector = '#continue';
+
+    firstNameSelector = '#first-name';
+
+    lastNameSelector = '#last-name';
+
+    zipCodeSelector = '#postal-code';
+
+    async clickContinue() {
+        await this.page.locator(this.continueSelector).click();
+    }
+
+    async fillInfo(info) {
```

Comment:
if your project will be written on pure JS - it will be better to use destructoring here, e.g.
```suggestion
    async fillInfo({ firstName, lastName, zipCode }) {
```
This will allow you easily read the code of the function and identify what exact object you need to pass here